session/bootstrap.go

```diff
@@ -2368,6 +2373,8 @@ func upgradeToVer135(s Session, ver int64) {
 
 	mustExecute(s, "INSERT HIGH_PRIORITY IGNORE INTO %n.%n VALUES (%?, %?);",
 		mysql.SystemDB, mysql.GlobalVariablesTable, variable.TiDBOptAdvancedJoinHint, false)
+
+	doReentrantDDL(s, CreateDefaultResourceGroup)
```

Comment:
Why do we add `CreateDefaultResourceGroup` to `upgradeToVer135`? Usually, when we add a new upgrade code, it is recommended to add a new `upgradeToVer...`.

Reply:
It's a result of rebase. L2374 is introduced in coming 7.0, which shares the version 135.